core/trino-server-rpm/src/main/resources/dist/etc/init.d/trino

```diff
@@ -25,45 +25,55 @@ SERVICE_USER='trino'
 # Use data-dir from node.properties file (assumes it to be at /etc/trino). For other args use defaults from rpm install
 NODE_PROPERTIES=/etc/trino/node.properties
 DATA_DIR=$(grep -Po "(?<=^node.data-dir=).*" $NODE_PROPERTIES | tail -n 1)
-SERVER_LOG_FILE=$(grep -Po "(?<=^node.server-log-file=).*" $NODE_PROPERTIES  | tail -n 1)
-LAUNCHER_LOG_FILE=$(grep -Po "(?<=^node.launcher-log-file=).*" $NODE_PROPERTIES  | tail -n 1)
-CONFIGURATION=(--launcher-config /usr/lib/trino/bin/launcher.properties --data-dir "$DATA_DIR" --node-config "$NODE_PROPERTIES" --jvm-config /etc/trino/jvm.config --config /etc/trino/config.properties --launcher-log-file "${LAUNCHER_LOG_FILE:-/var/log/trino/launcher.log}" --server-log-file "${SERVER_LOG_FILE:-/var/log/trino/server.log}")
+SERVER_LOG_FILE=$(grep -Po "(?<=^node.server-log-file=).*" $NODE_PROPERTIES | tail -n 1)
+LAUNCHER_LOG_FILE=$(grep -Po "(?<=^node.launcher-log-file=).*" $NODE_PROPERTIES | tail -n 1)
+CONFIGURATION=(
+    --launcher-config /usr/lib/trino/bin/launcher.properties
+    --data-dir "$DATA_DIR"
+    --node-config "$NODE_PROPERTIES"
+    --jvm-config /etc/trino/jvm.config
+    --config /etc/trino/config.properties
+    --launcher-log-file "${LAUNCHER_LOG_FILE:-/var/log/trino/launcher.log}"
+    --server-log-file "${SERVER_LOG_FILE:-/var/log/trino/server.log}"
+)
 
+# Don't need shellcheck to follow env.sh
+# shellcheck disable=1091
 source /etc/trino/env.sh
 
-start () {
+start() {
     echo "Starting ${SERVICE_NAME} "
-    if [ -z "${JAVA_HOME}" ]
-    then
-        echo "Warning: No value found for \$JAVA_HOME. Default Java will be used." >&2
+    if [ -z "${JAVA_HOME}" ]; then
+        # Don't expand JAVA_HOME in the error message
+        # shellcheck disable=2016
```

Comment:
Can this be on one line?

Reply:
No, but we could put it at the top to ignore all occurrences of this in the file: https://github.com/koalaman/shellcheck/wiki/Ignore

Reply:
let's avoid global / generic exclusions.